rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/rpginterop/rpg_system.kt

```diff
@@ -31,11 +32,33 @@ class DirRpgProgramFinder(val directory: File? = null) : RpgProgramFinder {
 
     override fun findRpgProgram(nameOrSource: String): RpgProgram? {
         val file = File(prefix() + nameAndSuffix(nameOrSource))
-        return if (file.exists()) {
-            RpgProgram.fromInputStream(FileInputStream(file), nameOrSource)
-        } else {
-            null
+
```

Comment:
This implementation should be reviewed (one day) because it contains too much duplicated code.
For instance if we add another extension we need to implement this thing through code duplication.

Reply:
ok